dxScheduler. TestCafe. Resize appointments and common code refactoring

naralewinsky opened the pull request:
Minimizing code by adding the same code in a loop of different dxScheduler views

Commit:
dxScheduler. TestCafe. Resize appointments and common code refactoring

Minimizing code by adding the same code in a loop of different dxScheduler views

churkin commented on testing/functional/tests/scheduler/resizeAppointments/timeline.ts:
```diff
@@ -0,0 +1,61 @@
+
+['timelineDay', 'timelineWeek', 'timelineWorkWeek'].forEach((mode) => test(`Resize in the "${mode}" mode`, async t => {
+	const scheduler = new Scheduler("#container");
+	const resizableAppointment = await scheduler.getAppointment('Brochure Design Review');
```
extra `await`

naralewinsky replied:
Complete.

Commit:
dxScheduler. TestCafe. Remove extra 'awaits' and beautify code

churkin commented on testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts:
```diff
@@ -0,0 +1,34 @@
+	{ id: "2", color: "#1e65e8" }
+];
+
+export const createScheduler = async (mode, dataSource, firstDayOfWeek = 1) => {
```
remove `async` and `await` please :)

naralewinsky replied:
Complete.

Commit:
dxScheduler. TestCafe. Remove extra 'awaits' in widget.data.ts

Commit:
dxScheduler. TestCafe. Remove extra 'async-awaits' in widget.data.ts

Commit:
TestCafe. dxScheduler. Fix "endTime" for resize in month mode

from '11pm' to '11am'

Commit:
dxScheduler. TestCafe. Fix "month" test

Olexi commented on testing/functional/tests/scheduler/dragAndDrop/basic.ts:
```diff
@@ -4,52 +4,28 @@ import url from '../../../helpers/getPageUrl';
 
 fixture `Drag-and-drop appointments in the Scheduler basic mode`
-    .page(url(__dirname, '../../container.html'));
+	.page(url(__dirname, '../../container.html'));
```
First 'tabulation' symbol must be 4 spaces, not 3 (as here).

Olexi commented on testing/functional/tests/scheduler/resizeAppointments/init/widget.data.ts:
```diff
@@ -0,0 +1,23 @@
+        text: 'Brochure Design Review',
+        startDate: new Date(2019, 3, 1, 10, 0),
+        endDate: new Date(2019, 3, 1, 11, 0),
+        resourceId: '0'
```
resourceId must be `number`, not `string`

Olexi commented on testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts:
```diff
@@ -0,0 +1,33 @@
+export const currentDate = new Date(2019, 3, 1);
+
+export const resources = [
+	{ id: "0", color: "#e01e38" },
```
id must be `number`

Olexi commented on testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts:
```diff
@@ -0,0 +1,33 @@
+import { createWidget } from "../../../../helpers/testHelper";
+
+export const widgetHeight = 833;
```
`const` should not be separated from the initialization of Scheduler, 'cause of single-use of it

Commit:
dxScheduler. TestCafe. Some improvements in the code

The problem with tabulation (replaced by spaces), exported variables in the widget (extra code), replacing the resource identifier from text to numeric type and other changes has been resolved.

Commit:
dxScheduler. TestCafe. Minor changes.

Commit:
Change dimensions width <-> height
